CMakeLists.txt

```diff
@@ -217,8 +223,14 @@ find_package(SDL2_mixer REQUIRED)
 
 find_package(PNG REQUIRED)
 
-find_package(yaml-cpp REQUIRED HINTS /usr/lib32/cmake/yaml-cpp)
-include_directories(${YAML_CPP_INCLUDE_DIR})
+find_package(yaml-cpp CONFIG REQUIRED)
```

Comment:
I believe the hints were added for a reason, would be good to leave them in, as it's likely going to break compilation on our machines

Reply:
Yeah, the macOS CI is failing because of this change. While it can be argued that the path should just be passed to CMake, I think it would be better to just revert this change.